src/index.js

```diff
@@ -30,13 +35,15 @@ export default function supernova(env) {
       const layout = useStaleLayout();
       const model = useModel();
       const constraints = useConstraints();
+      const translator = useTranslator();
       const selectionsAPI = useSelections();
       const theme = useTheme();
 
       const [pageInfo, setPageInfo] = useState(() => ({ top: 0, height: 100 }));
       const [muiParameters] = useState(() => muiSetup());
       const [tableData] = usePromise(() => manageData(model, layout, pageInfo), [layout, pageInfo]);
 
+      locale(translator);
```

Comment:
Should tie this to language changes as well.

Reply:
Good point. How can it know that language is changed? which API could I use?

Reply:
is translator.language()?

Reply:
fixed